Assets/Scripts/Models/Character.cs

```diff
@@ -357,6 +366,13 @@ public void WriteXml(XmlWriter writer)
         }
 
         writer.WriteAttributeString("needs", needString);
+
+        StringBuilder statsString = new StringBuilder();
+        foreach (KeyValuePair<string, Stat> stat in stats)
+        {
+            statsString.Append(stat.Value.statType).Append(";").Append(stat.Value.Value).Append(":");
+        }
+        writer.WriteAttributeString("stats", statsString.ToString());
```

Comment:
I was following the same style that the Needs were using, thus why I went with that way.  I do agree that it is cleaner to save them as separate elements like that.. I can certainly make that change... but unless it is desired, i would prefer not to touch needs unless I have to, but it will be inconsistent if I don't.